app/modules/asset/service.server.ts

```diff
@@ -540,6 +540,46 @@ export async function deleteNote({
   });
 }
 
+export async function duplicateAsset({
+  asset,
+  userId,
+  amountOfDuplicates,
+}: {
+  asset: Prisma.AssetGetPayload<{ include: { custody: true } }>;
```

Comment:
I never did it like this but I really like it. Thanks for teaching me something. I will use this `...GetPayload` type from now on